@navikt/core/icons/meta.json

```
@@ -737,6 +737,12 @@
     "created_at": "2020-07-01T10:37:43.021Z",
     "pageName": "Status"
   },
+  {
+    "name": "ErrorColor",
```

Er dette navnet hakket bedre?
```suggestion
    "name": "ErrorColored",
```

---

Ja syntes det var bedre. Må fikses i Figma da evt før ny synk.